Automattic/sensei
# Export lessons CSV

Fixes #3049

### Changes proposed in this Pull Request

* Introduces the lesson exporter task.

### Testing instructions

* Create courses, lessons, and questions, tying all of them. Try to cover the maximum edge cases while creating the posts (each post with different options).
* Open exporter, select all options (courses, lessons, and questions), and click `Export`.
* Check the exported file and make sure it matches the data you created.
--
* Delete all courses/switch to an empty test site.
* Import the exported files.
* All details should be properly imported.
